pages/api/doctor/api.ts

```diff
@@ -0,0 +1,55 @@
+// Next.js API route support: https://nextjs.org/docs/api-routes/introduction
+import type { NextApiRequest, NextApiResponse } from 'next'
+import { prisma } from "../../../lib/prisma";
+
+type Data = {
+  name: string
+}
+
+export default async function handler(
+  req: NextApiRequest,
+  res: NextApiResponse<Data | null>
+) {
+
+if(req.method === 'POST'){
+
+try {
+  const{name,email,passwordHash, userId, crm, crmUf} = req.body;
+
+
+  const user = await prisma.user.create({
+    data:{
+      id: undefined, //pensando em como pegar esse id
+      name,
+      email,
+      passwordHash,
+      role: undefined,
+      createdAt: Date(),
+      updatedAt: Date(),
+      admin: undefined,
+      doctor: undefined   
+    }
+  });
+
+  
+
+  const doctor = await prisma.doctor.create({

```

Comment:
You can let prisma create nested resources. check here: https://www.prisma.io/docs/concepts/components/prisma-client/relation-queries#create-a-related-record